src/haz3lschool/SchoolExercise.re

```diff
@@ -668,6 +668,18 @@ module F = (ExerciseEnv: ExerciseEnv) => {
     };
   };
 
+  let syntax_test = ({eds, _}: state): SyntaxTest.SyntaxReport.t => {
+    let user_impl_term =
+      Util.TimeUtil.measure_time("user_impl_term_syntax", true, () =>
+        EditorUtil.stitch([eds.your_impl])
+      );
+    let syntax_params: SyntaxTest.params = {
+      var_mention: [("xyz", 50.), ("ext", 50.)],
+      recursive: [("odd", 100.), ("dne", 0.)],
+    };
+    SyntaxTest.check(user_impl_term, syntax_params);
```

Comment:
these are hard-coded right now, they should be in the SchoolExercise data structure so that each exercise can specify different syntax tests